README.md

```diff
@@ -6,6 +6,13 @@ For official StackStorm Docker repo, go https://github.com/StackStorm/st2-docker
 Interested in StackStorm HA in Kubernetes? Check https://github.com/stackstorm/stackstorm-ha
 
 This repository contains Dockerfiles, used by StackStorm team for CI and some other specific needs.
+
+## `buildpack`
+
+`Dockerfiles` with pre-installed requirements to build `.deb` and `.rpm` StackStorm packages.
+
+See [buildpack/](buildpack/)
+
```

Comment:
```suggestion
```

Buildpack is not used anywhere anymore.
Once we moved ldap and rbac as direct `st2` requirements instead of building deb/rpm enterprise packages for them these Dockerfiles became obsolete.

It's a great reminder now to clean up that `buildpack` dir as well.

Reply:
Done